docs/troubleshooting.rst

```diff
@@ -32,6 +32,32 @@ General
    
   Try running ``yarn run build-less`` or ``npm run build-less`` from the project directory.
 
+* ``[/usr/bin/env: 'bash\r': No such file or directory] or [exec scripts/wait-for-postgres.sh: no such file or directory]``
+
+  These errors may occur when running develop.sh or database-init-docker.sh during installation if git or another application
+  has replaced Linux line endings (LF) with Windows line endings (CR LF) in certain .sh files.
+
+  To determine if this is the issue, you may open the .sh files using an application such as Notepad++ or VS Code and check
+  the bottom right corner. If any file's line endings are indicated incorrectly as Windows (CR LF) you can resolve the issue
+  by following these steps:
+
+  1. Navigate to the bookbrainz-site directory and configure git to clone repositories using the original line endings
+  with the following command:
+  ::
+
+     git config --global core.autocrlf input
+
+  2. Delete the bookbrainz-site directory from its parent directory.
+  ::
```

Comment:
This `::` is used to present a block of code or a command, and is not required here
```suggestion

```